### PR TITLE
docs: Update/Add documentation

### PR DESCRIPTION
**Pull request by ynotdraw**
### Changes

- Fix formatted block
- Add `Development` section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To get started with Chroma, follow these steps:
    import { ThemeProvider } from '@lifeomic/chroma-react/styles';
 
    function App({ children }) {
-    return <ThemeProvider>{children}</ThemeProvider>;
+     return <ThemeProvider>{children}</ThemeProvider>;
```
Comment (ynotdraw) [inline README.md:35]: Here and up is auto-formatted changes

File: README.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ To get started with Chroma, follow these steps:
 
 1. Install dependencies
 
-    ```bash
-    yarn add @lifeomic/chroma-react @material-ui/styles react-router-dom
-    ```
+   ```bash
+   yarn add @lifeomic/chroma-react @material-ui/styles react-router-dom
+   ```
```
Comment (ynotdraw) [inline README.md:25]: Here down are auto-formatted changes

File: README.md [hunk 3]
```diff
@@ -60,60 +60,107 @@ Want to override the default theme of Chroma? No problem!
...
-    Need to build a custom component, but want to use the styles hook of an existing Chroma component?
+Need to build a custom component, but want to use the styles hook of an existing Chroma component?
+
+```jsx
+import { useStyles } from '@lifeomic/chroma-react/components/Button/Button';
+
+const CustomButton = ({}) => {
+  const classes = useStyles({});
+  return <button className={classes.root}>Custom Button</button>;
+};
```
Comment (ynotdraw) [inline README.md:117]: This section in particular had a bad backtick block due to the spacing, so it's fixed with this.

**Review by dexterca — APPROVED** [review]

**Review by smolster — APPROVED** [review]
🦸🏻

**Pull request merged**